lib-multisrc/heancms/src/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -100,7 +100,7 @@ abstract class HeanCms(
 
         val response = client.newCall(POST("$apiUrl/login", headers, body)).execute()
 
-        if (!response.isSuccessful) return null
+        if (!response.isSuccessful) throw Exception(intl["invalid_credentials_error"])
```

Comment:
I was think throw the actual error from api;
```
result.errors?.let { errors ->
    val message = errors.firstOrNull()?.message ?: intl["login_failed_unknown_error"]
    
    throw Exception(message)
}
```

Reply:
another DTO if response is not successful or add 'errors' to 'HeanCmsTokenPayloadDto' and throw if exists?